logic/love_count.py

```diff
@@ -22,11 +22,12 @@ def top_lovers_and_lovees(utc_week_start, dept=None, limit=20):
         if c.sent_count == 0:
             continue
         employee_key = c.key.parent()
-        if dept:
-            employee = employee_key.get()
-            if employee.meta_department == dept or employee.department == dept:
-                lovers.append((employee_key, c.sent_count))
-        else:
+        employee = employee_key.get()
+        if (
```

Comment:
Not blocking but I think this could need a comment saying what condition we are looking for to append an employee to the list.

Reply:
Maybe even better to extract this into a function with a little docstring to keep things DRY. Thoughts?